my-app/src/components/calendar/calendar.css

```diff
@@ -0,0 +1,25 @@
+@import '~@fullcalendar/core/main.css';
+@import '~@fullcalendar/daygrid/main.css';
+@import '~@fullcalendar/timegrid/main.css';
```

Comment:
I can't run your build because it doesn't compile what did you install for the calendar?

Reply:
@rbhalodi I was able to run the build but I did 'npm i' in the project directory and in the my-app directory to make sure all dependencies are installed. I'm not sure if this will fix your issue but it's smart to do this whenever you checkout a new branch.

Reply:
ohh got it working thanks!